### ISSUE TITLE
HB timeout when connecting to deconz, deconz reachable on network and working

### ISSUE THREAD

**Issue by ndx1905-github** [ebaauw/homebridge-hue#927]
#### Issue

I use Homebridge in a docker (oznu/homebridge alpine version) that can be reached from localnetwork at 192.168.1.91:8581
I use phoscon/conbee/deconz in another docker that can be reached from localnetwork at 192.168.1.91:1090

Deconz is fully accessible and working as a web page + manual API requests are working perfectly too. 

I've installed Homebridge-hue plugin which
- works with a standalone philips hue bridge on the network (192.168.1.41)
- but does a time out on the deconz server (192.168.1.91:1090). There is no 404 error like if there was no response at all, just a time out. Again, if I try to connect to 192.168.1.91:1090 myself with the browser or with a rest client, deconz works normally.

#### Log Messages

[2/13/2021, 1:26:11 PM] [HB Supervisor] Starting Homebridge with extra flags: -I -P /homebridge/node_modules
[2/13/2021, 1:26:11 PM] [HB Supervisor] Started Homebridge v1.2.5 with PID: 6149
[2/13/2021, 1:26:11 PM] Loaded config.json with 0 accessories and 2 platforms.
[2/13/2021, 1:26:11 PM] ---
[2/13/2021, 1:26:12 PM] Loaded plugin: homebridge-dummy@0.5.0
[2/13/2021, 1:26:12 PM] Registering accessory 'homebridge-dummy.DummySwitch'
[2/13/2021, 1:26:12 PM] ---
[2/13/2021, 1:26:13 PM] Loaded plugin: homebridge-hue@0.12.22
[2/13/2021, 1:26:13 PM] Registering platform 'homebridge-hue.Hue'
[2/13/2021, 1:26:13 PM] ---
[2/13/2021, 1:26:13 PM] Loaded plugin: homebridge-config-ui-x@4.38.0
[2/13/2021, 1:26:13 PM] Registering platform 'homebridge-config-ui-x.config'
[2/13/2021, 1:26:13 PM] ---
[2/13/2021, 1:26:13 PM] Loading 2 platforms...
[2/13/2021, 1:26:13 PM] [Config] Initializing config platform...
[2/13/2021, 1:26:13 PM] [Config] Running in Service Mode
[2/13/2021, 1:26:13 PM] [Hue] Initializing Hue platform...
[2/13/2021, 1:26:13 PM] [Hue] homebridge-hue v0.12.22, node v14.15.5, homebridge v1.2.5
[2/13/2021, 1:26:18 PM] [Hue] 192.168.1.91: request 1: GET /config
[2/13/2021, 1:26:18 PM] [Hue] 192.168.1.91: request 1: 192.168.1.91: timeout after 5 seconds
[2/13/2021, 1:26:18 PM] [Hue] 192.168.1.91:1080: not found - retrying in 30s
[2/13/2021, 1:26:33 PM] [Hue] homebridge-hue is taking a long time to load and preventing Homebridge from starting.
[2/13/2021, 1:26:53 PM] [Hue] homebridge-hue is taking a long time to load and preventing Homebridge from starting.
[2/13/2021, 1:26:53 PM] [Hue] 192.168.1.91: request 1: GET /config
[2/13/2021, 1:26:53 PM] [Hue] 192.168.1.91: request 1: 192.168.1.91: timeout after 5 seconds
[2/13/2021, 1:26:53 PM] [Hue] 192.168.1.91:1080: not found - retrying in 30s
[2/13/2021, 1:27:13 PM] [Hue] homebridge-hue is taking a long time to load and preventing Homebridge from starting.

#### Debug Files

[homebridge-hue.json.gz](https://github.com/ebaauw/homebridge-hue/files/5975674/homebridge-hue.json.gz)

**Comment by ab1781** [comment]
The same here, deconz is working with HB, but there is always a loooong delay, most of all in automations, e.g. light on with motion sensor. I switched back to 12.21, thanks for fixing in advance.

**Comment by ndx1905-github** [comment]
I guess it’s different. When I mean « timeout », I mean that it waits and waits and then never connects. 
I don’t mean  that it works slowly or with a delay like you describe. 
It’s a different situation I think.

**Comment by ab1781** [comment]
Maybe, maybe not ;-). I should have added this: In the first time nothing worked here, so no lag but obviously a time out. I tested the automations then in HomeKit manually, which somehow brought them back to life, but as noticed only slowww. Haven't got time to dive in deeper for now, but it seems there is clearly something in the new version wrong. Any ideas what?

**Comment by ebaauw** [comment]
> Again, if I try to connect to 192.168.1.91:1090 myself with the browser or with a rest client, deconz works normally.

Did you try this from the container running Homebridge?

**Comment by ndx1905-github** [comment]
> > Again, if I try to connect to 192.168.1.91:1090 myself with the browser or with a rest client, deconz works normally.
> 
> Did you try this from the container running Homebridge?

WoW thank you for that question, it really opened my eyes. So I hadn't tried, and indeed I couldn't reach the deconz container from the terminal inside the homebridge container.
I had to use the docker network ip and ports (172.xxx instead of 192.xx). Since those addresses are not fixed but change after a reboot, I had to create a new bridge to be able to open address the deconz container by name. See here https://jakupovic.org/wiki/syno-docker-nextcloud/ to see how it works.

Thanks for putting me on the right path. it all works now :)
